resources/styles/loop.less

```diff
@@ -553,6 +553,74 @@ label[for=mw-fr-commentbox] {
 	background-color: @content-outter-background !important;
 }
 
+/** *** *** *** *** *** *** *** *** *** *** **/
+/**                LOOP AREA                **/
+/** *** *** *** *** *** *** *** *** *** *** **/
+
+.looparea {
+    width: calc(~"100% + 131px"); // magic number
+    left: -100px;
+	position: relative;
+	clear: both;
+}
+
+.looparea-container {
+    display: flex;
+}
+
+.looparea-left {
+    min-width: 85px;
+    max-width: 85px;
+    text-align: center;
+    border-right: 3px solid @looparea-color;
+    font-size: 0.8rem;
+    position: relative;
+}
+
+.looparea-right {
+    background: @looparea-background;
+    width: 100%;
+    margin-right: 15px; /* class m-3 is missing 1-2 pixels */
+}
+
+.looparea [class^="ic-"], .looparea [class*=" ic-"] {

```

Comment:
Wenn Loop Objekte in Areas sind, werden die ebenfalls erfasst und 2rem groß. daher bitte ändern zu
.looparea-left span.ic {